team_production_system/models/availability.py

```diff
@@ -5,16 +5,28 @@
 
 # Allow mentors to set their avaliabiltiy
 class Availability(models.Model):
+
+    STATUS_CHOICES = [
+        ('Open', 'Open'),
+        ('Requested', 'Requested'),
+        ('Confirmed', 'Confirmed'),
+    ]
+
     mentor = models.ForeignKey(
         Mentor, on_delete=models.CASCADE, related_name='mentor_availability')
     start_time = models.DateTimeField()
     end_time = models.DateTimeField()
+    status = models.CharField(
+        max_length=10, choices=STATUS_CHOICES, default='Open')
```

Comment:
Nit-pick: Each argument should get it's own line, as well as the closing parenthesis